test/intl402/Temporal/TimeZone/prototype/getNextTransition/transition-at-instant-boundaries.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.timezone.prototype.getnexttransition
+description: >
+  Test transitions at the instant boundaries.
+features: [Temporal, Intl-enumeration]
+---*/
+
+const min = new Temporal.Instant(-86_40000_00000_00000_00000n);
+const max = new Temporal.Instant(86_40000_00000_00000_00000n);
+
+for (let id of Intl.supportedValuesOf("timeZone")) {
+  let tz = new Temporal.TimeZone(id);
+
+  // If there's any next transition, it should be after |min|.
+  let next = tz.getNextTransition(min);
+  if (next) {
+    assert(next.epochNanoseconds > min.epochNanoseconds);
+  }
+
+  // There shouldn't be any next transition after |max|.
+  next = null;
+  try {
+    next = tz.getNextTransition(max);
+  } catch (e) {
+    // This can throw a RangeError.
```

Comment:
This is no longer the case, I think?

Reply:
<https://github.com/tc39/proposal-temporal/pull/2351> will ensure no error is thrown.

Reply:
I made that change and moved this into #3610.